### ISSUE TITLE
Accessibility: Cannot Publish Post Easily Using Mac OS X and Safari

### ISSUE THREAD

**Issue by davidakennedy** [Automattic/wp-calypso#21218]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/post/site/post-ID
2. Turn on Voiceover.
3. Enter a post title and some text in the post area.
4. Exit from the text area and navigate to the Publish button.
5. Next, try to hit the "green" Publish button.

#### What I expected

I expected that my focus would be shifted to the "green" publish button, but it's not. Making it hard to publish a post, because then I have to navigate so many elements within a frame, form and the like to find the right button.

#### What happened instead

It looks like my focus is shifted, but when I press click again on the button, it brings up the post history. So my focus needs to be shifted to the right element. As soon as I click, I get lost because of all the things on the page.

#### Browser / OS version

Safari 11.0.2/Mac OS 10.13.2 

#### Screenshot / Video

https://cloudup.com/cL7O0OOnj2D

#### Context / Source

Found via #user-report `#790182-zen`

**Comment by rachelmcr** [comment]
Related: https://github.com/Automattic/wp-calypso/issues/1998

**Comment by kriskorn** [comment]
A new user report in #1436732-zen. They also have the same issue in WP Admin, so this might be a core issue as well. Will clarify about the latter.
